v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -159,28 +155,39 @@ func AddSubcommand(app *clir.Cli, w io.Writer) error {
 			return err
 		}
 
-		// Run go mod tidy to ensure we're up to date
+		// Run go mod tidy to ensure we're up-to-date
 		err = runCommand(cwd, false, "go", "mod", "tidy", "-compat=1.17")
 		if err != nil {
 			return err
 		}
 
+		buildOptions := generateBuildOptions(flags)
+		buildOptions.Logger = logger
+
 		if !flags.noGen {
 			self := os.Args[0]
-			if flags.tags != "" {
-				err = runCommand(cwd, true, self, "generate", "module", "-tags", flags.tags)
+			var env []string
+			if flags.obfuscate {
+				env = append(env, "WAILS_OBFUSCATE=true")
+			}
+
+			userTags, err := common.ParseUserTags(flags.tags)
+			if err != nil {
+				return err
+			}
+			if len(userTags) > 0 {
+				buildOptions.UserTags = userTags
+				genModuleTags := lo.Without(userTags, "desktop", "dev", "debug")
```

Comment:
This seems to be mostly redundant with the code of the module command itself in `module.go` line 44. Should we remove this here and also exclude `dev` in `module.go`, so we are doing this just in one place?